hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/protocol/commands/ReplicateContainerCommand.java

```diff
@@ -41,20 +41,28 @@
 
   private final long containerID;
   private final List<DatanodeDetails> sourceDatanodes;
+  private final long timeoutMs;
```

Comment:
If we go with my earlier suggestion of setting the "run by time" in SCM on the command, perhaps rename "timeout" to something like "deadlineEpochMs" or "expiryEpochMs" throughout this method.